design/RejectedErrorHandling.md

```diff
@@ -0,0 +1,56 @@
+Appendix document for [ErrorHandling.md](ErrorHandling.md).
```

Comment:
Note: this is a rejected proposal listed here for posterity.

Reply:
Added this to the top of the doc